src/main/java/io/github/thebusybiscuit/slimefun4/utils/SlimefunUtils.java

```diff
@@ -97,23 +112,25 @@ public static boolean isSoulbound(@Nullable ItemStack item) {
             SlimefunItem sfItem = SlimefunItem.getByItem(item);
 
             if (sfItem instanceof Soulbound) {
-                return !sfItem.isDisabled();
+                if (world != null) {
+                    return !sfItem.isDisabled() && !sfItem.isDisabledIn(world);
```

Comment:
I think (you may wanna double-check this) this can be simplified.

```suggestion
                    return !sfItem.isDisabledIn(world);
```